src/utils/date-time.js

```diff
@@ -77,33 +78,52 @@ export default {
     return new Date(d.getFullYear(), d.getMonth(), 1);
   },
 
-  getWeekArray(d) {
+  getFirstDayOfWeek() {
+    const now = new Date();
+    return new Date(now.setDate(now.getDate() - now.getDay()));
+  },
+
+  getWeekArray(d, firstDayOfWeek = 0) {
```

Comment:
Can you set the default value in the `getDefaultProps` of the `DateTimePicker` component?
This will automatically be used by the documentation.